src/components/workflows/WorkflowTable.js

```diff
@@ -164,6 +169,21 @@ function DetailButton(props) {
   );
 }
 
+function TableCellData(props) {
+  const { obj_item, curr_key } = props;
+  if (curr_key === "end_status") {
+    return <StatusChip status={obj_item[curr_key]} />;
+  }
+
+  // Converting time to be more readable
+  if ((curr_key === "start") | (curr_key === "end")) {
+    const date_utc = new Date(obj_item[curr_key]);
+    return date_utc.toLocaleString("en-GB", { timeZone: "UTC" });
```

Comment:
Just minor. Would be good to use Moment wrap here to show as local time, instead of UTC.